src/components/icons/OutdentIcon.tsx

```diff
@@ -0,0 +1,88 @@
+import { icon } from '../../../styled-system/recipes'
+import { token } from '../../../styled-system/tokens'
+import IconType from '../../@types/Icon'
+
+// eslint-disable-next-line jsdoc/require-jsdoc, react-refresh/only-export-components
```

Comment:
The `react-refresh/only-export-components` rule can be re-enabled safely, and we should be able to remove `jsdoc/require-jsdoc` by adding a simple JSDoc comment here.